### DISCUSSION TITLE
Key Bind to open floating pane plus resize it.

### DISCUSSION THREAD

**Discussion by gluiz1** [zellij-org/zellij#3573] [Q&A] [answered]
I want to create a unique key bind to open a floating pane and then resize it. My solution for now is:

```
keybinds {
    normal {
        bind "Ctrl k" {ToggleFloatingPanes;}
        bind "Ctrl j" {SwitchToMode "Resize"; Resize "Increase"; Resize "Increase"; Resize "Increase"; Resize "Increase"; Resize "Increase"; Resize "Increase"; Resize "Increase"; SwitchToMode "Normal";}
    }
}
```

Both work as expected. But there is a problem  when I try to mix then, for example, I tried to put all commands in one key bind:

```
keybinds {
    normal {
        bind "Ctrl k" {ToggleFloatingPanes; SwitchToMode "Resize"; Resize "Increase"; Resize "Increase"; Resize "Increase"; Resize "Increase"; Resize "Increase"; Resize "Increase"; Resize "Increase"; SwitchToMode "Normal";}
    }
}
```

With this definition, firstly `zellij` opens a floating pane with normal size and then, when I press the key bind a second time, `zellij` resizes it. 

What I want: 

I want to press the key bind and get the floating pane resized at the first stage. Is this possible? If yes, what I am missing here?

**Comment by imsnif** [comment, marked as answer]
You can kind of do it like this (replace 'bash' with your shell of choice):
```
bind "n" { Run "bash" { floating true; x "12%"; y "12%"; width "75%"; height "75%"; }; SwitchToMode "normal"; }
```

Otherwise you can also do this through the CLI: 
```
zellij action new-pane --floating --width 75% --height 75% --x 12% --y 12%
```

The `NewPane` keybinding has unfortunately not caught up yet.
